cmd/podman/shared/create.go

```diff
@@ -483,6 +483,16 @@ func ParseCreateOpts(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 
 	// ENVIRONMENT VARIABLES
 	env := EnvVariablesFromData(data)
+	if c.Bool("env-host") {
+		for _, e := range os.Environ() {
+			pair := strings.SplitN(e, "=", 2)
+			if _, ok := env[pair[0]]; !ok {
```

Comment:
Shouldn't we overwrite if the goal is to send over the entire host environment?

Reply:
No I believe the precidence should be from the host, image, env-file, env in that order.  That is what I documented in the man page.